src/go/rdctl/pkg/paths/paths_darwin.go

```diff
@@ -48,3 +51,38 @@ func GetPaths(getResourcesPathFuncs ...func() (string, error)) (Paths, error) {
 
 	return paths, nil
 }
+
+// Return the path used to launch Rancher Desktop.
+func GetRDLaunchPath(ctx context.Context) (string, error) {
+	appDir, err := directories.GetApplicationDirectory(ctx)
+	if err != nil {
+		return "", fmt.Errorf("failed to get application directory: %w", err)
+	}
+	executablePath := []string{"Contents", "MacOS", "Rancher Desktop"}
+
+	for _, dir := range []string{appDir, "/Applications/Rancher Desktop.app"} {
+		absPathParts := append([]string{dir}, executablePath...)
+		ok, err := checkUsableApplication(filepath.Join(absPathParts...), true)
+		if err != nil {
+			return "", err
+		}
+		if ok {
+			return dir, nil
+		}
+	}
+	return "", errors.New("search locations exhausted")
```

Comment:
Should the returned error be more descriptive to include the locations that were searched (we will thank ourselves during debugging :b)?